src/client.js

```diff
@@ -0,0 +1,155 @@
+import debug from 'debug'
+import {
+  on,
+} from 'events'
+import graphql from 'graphql'
+import which_stream from './stream.js'
+import {
+  promisify,
+} from 'util'
+import WebSocket from 'ws'
+import invariant from 'invariant'
+
+const log = debug('gql-ws').extend('client')
+const {
+  stripIgnoredCharacters, parse,
+} = graphql
+const DEFAULT_TIMEOUT = 31_000
+const util_promisify =
```

Comment:
Why when you have `promisify` from util ?

Reply:
in the browser ?

Reply:
Webpack will polyfill it